index.js

```diff
@@ -40,3 +46,66 @@ const popupBtn = document.getElementById("btn-popup"); // Close popup button
 
 //TODO: Add event listeners to the buttons
 //code here
+
+
+// EVENT LISTENERS FOR BUTTONS 
+
+body.addEventListener("click", function(e){	
+	if (e.target.id === "feed-btn"){
+		addToPetCareArray("food")
+	} else if (e.target.id === "play-btn"){
+		addToPetCareArray("play")
+	} else if (e.target.id === "groom-btn"){
+		addToPetCareArray("groom")
+	} else if (e.target.id === "reset-btn"){
+		resetGame()
+	} else if (e.target.id === "cyp-sendbtn"){
+		renderMainApp()
+    }
+})
```

Comment:
Wouldn't it be better to split it into sections for readability? 
```suggestion
const actionBtns = document.querySelector(".action-btns")

actionBtns.addEventListener("click", function(e){	
	if (e.target.id === "feed-btn"){
		addToPetCareArray("food")
	} else if (e.target.id === "play-btn"){
		addToPetCareArray("play")
	} else if (e.target.id === "groom-btn"){
		addToPetCareArray("groom")
	} 
	
	
})

//event listener for section containing reset button
//event listener for CYP section
```